firmware_motion_board/src/control_system.c

```diff
@@ -0,0 +1,135 @@
+#include "control_system.h"
+
+volatile PidReg_t reg_left = PIDREG_DEFAULTS;
+volatile PidReg_t reg_right = PIDREG_DEFAULTS;

```

Comment:
Samo neka polja iz strukture su volatile?

Reply:
Uklonio sam volatile odatle i unutar strukture sam neke proglasio za volatile, to proveriti da li je ok.

Reply:
Ja ne vidim volatile nigdje sad :/

Reply:
Nalazi se unutar structa.